.eslintrc

```diff
@@ -23,6 +24,8 @@
     "jsx-quotes"           : [2, "prefer-single"],
     "max-len"              : [2, 120, 2],
     "object-curly-spacing" : [2, "always"],
-    "comma-dangle"         : "off"
+    "comma-dangle"         : "off",
+    "no-trailing-spaces"   : "off",
```

Comment:
The second one stopped me from injecting some air in my code. I think this leaded to less-readable code in some places